src/utils/downloads.ts

```diff
@@ -0,0 +1,27 @@
+const filesToDownload = [
+	{
+		blob: new URL("../../public/marker/marker_file.pdf", import.meta.url).href,
```

Comment:
```suggestion
		blob: new URL("/marker/marker_file.pdf", import.meta.url).href,
```
これでいけるくない？
(publicからインポートすることは基本ないと思ってもらえると)

Reply:
`blob: new URL("/marker/marker_file.pdf", import.meta.url).href,`

修正しました！